src/test/java/org/alfresco/RepositoryStartStopTest.java

```diff
@@ -187,9 +186,6 @@ public void ignoredTestOpenCloseOpenCloseFull() throws Exception
        ctx = getFullContext();
        assertEquals(ctx, ctx2);
        
-       // Refresh it, shouldn't break anything
-       ((AbstractApplicationContext)ctx).refresh();

```

Comment:
Erik, I would rather prefer to keep these lines here - commented
and also add a java comment linking to https://issues.alfresco.com/jira/browse/REPO-4176 
I also think that enabling this context refresh should be part of the acceptance criteria on that repo ticket.